sharktank/sharktank/evaluate/perplexity.py

```diff
@@ -136,7 +136,7 @@ def get_prompts(self):
             "text"
         ]
 
-        num_test_prompts = 250
+        num_test_prompts = 219
```

Comment:
Why this value and how long does it take to run? Since this is a nightly, I assume we can run a much larger set of prompts if not the whole dataset

Reply:
I want at least 100 prompts to run for nightly. This is num_prompts before filtering to get to 100 prompts post filtering. At 100 prompts, 8B runs in 20 mins and 405B took 9 hrs. So 405B will take most of the time of nightly, if we want to free up 30F for dev in the morning.